integrations/tensorflow/bindings/python/pyiree/tf/compiler/BUILD

```diff
@@ -51,6 +51,20 @@ config_setting(
 # See: https://github.com/google/iree/issues/1506
 SAVED_MODEL_TF_RUNTIME_DEPS = [
     "@org_tensorflow//tensorflow/core:ops",
+
+    # Deps for MobileBert
```

Comment:
please explicitly specify that we would like to depend on just the big tensorflow target, but we pare it down for build time concerns, and provide numerical examples of the build time advantage. Mention that this is only needed for SignaureDef SavedModels which require a Session and running some ops.

There isn't anything specific to MobileBert here; just mention that these are needed for our current set of SignatureDef SavedModels.

Also, as a TODO, maybe consider isolating SignatureDef SavedModel functionality into their own library so only tests using SignatureDef SavedModel's pay this build time cost.

Reply:
Done. I'd also like to move the mobile bert deps into that target, but I'm unsure of the details of how to actually do that.